cmd/gotk/get_helmrelease.go

```diff
@@ -51,37 +54,43 @@ func getHelmReleaseCmdRun(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Name", "Revision", "Ready Status", "Message", "Namespace"})
+	table.SetRowLine(true)
+
 	var list helmv2.HelmReleaseList
 	err = kubeClient.List(ctx, &list, client.InNamespace(namespace))
 	if err != nil {
 		return err
 	}
 
 	if len(list.Items) == 0 {
-		logger.Failuref("no releases found in %s namespace", namespace)
+		table.Append([]string{"None", "None", "Unknown", "No releases found", namespace})
+		table.Render()
 		return nil
 	}
 
 	for _, helmRelease := range list.Items {
 		if helmRelease.Spec.Suspend {
-			logger.Successf("%s is suspended", helmRelease.GetName())
+			table.Append([]string{helmRelease.GetName(), helmRelease.Status.LastAppliedRevision, "false", "suspended", namespace})
 			continue
 		}
 		isInitialized := false
 		if c := meta.GetCondition(helmRelease.Status.Conditions, meta.ReadyCondition); c != nil {
 			switch c.Status {
 			case corev1.ConditionTrue:
-				logger.Successf("%s last applied revision %s", helmRelease.GetName(), helmRelease.Status.LastAppliedRevision)
+				table.Append([]string{helmRelease.GetName(), helmRelease.Status.LastAppliedRevision, "true", "ready", namespace})
 			case corev1.ConditionUnknown:
-				logger.Successf("%s reconciling", helmRelease.GetName())
+				table.Append([]string{helmRelease.GetName(), helmRelease.Status.LastAppliedRevision, "unknown", "reconciling", namespace})
 			default:
-				logger.Failuref("%s %s", helmRelease.GetName(), c.Message)
+				table.Append([]string{helmRelease.GetName(), helmRelease.Status.LastAppliedRevision, "false", c.Message, namespace})
 			}
 			isInitialized = true
 		}
 		if !isInitialized {
-			logger.Failuref("%s is not ready", helmRelease.GetName())
+			table.Append([]string{helmRelease.GetName(), helmRelease.Status.LastAppliedRevision, "false", "not ready", namespace})
 		}
```

Comment:
I think this can be reduced to (this should be applicable to all other `get` commands, with some minor variable name changes):

```suggestion
		if c := meta.GetCondition(helmRelease.Status.Conditions, meta.ReadyCondition); c != nil {
			table.Append([]string{
				helmRelease.GetNamespace(),
				helmRelease.Status.LastAppliedRevision,
				string(c.Status),
				c.Message,
				strings.Title(strconv.FormatBool(helmRelease.Spec.Suspend)),
			})
		} else {
			table.Append([]string{
				helmRelease.GetName(),
				helmRelease.Status.LastAppliedRevision,
				"False",
				"waiting to be reconciled",
				strings.Title(strconv.FormatBool(helmRelease.Spec.Suspend)),
			})
		}
```

Reply:
Did you mean to have `helmRelease.GetNamespace()` and `helmRelease.GetName()` differ in the if/else here?

Reply:
I indeed made a typo, good catch :+1: